homeassistant/components/flipr/coordinator.py

```diff
@@ -6,39 +6,44 @@
 from flipr_api import FliprAPIRestClient
 from flipr_api.exceptions import FliprError
 
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
-from .const import CONF_FLIPR_ID
-
 _LOGGER = logging.getLogger(__name__)
 
 
-class FliprDataUpdateCoordinator(DataUpdateCoordinator):
-    """Class to hold Flipr data retrieval."""
+class BaseDataUpdateCoordinator(DataUpdateCoordinator):
+    """Parent class to hold Flipr and Hub data retrieval."""
 
```

Comment:
```suggestion

    config_entry: ConfigEntry

```
Overwriting the types saves you from the if statement in your base entity, because this is a config flow based integration, so this can't be None

Reply:
OK, done.